asapdiscovery-docking/asapdiscovery/docking/cluster_scripts/run_fragalysis_retrospective_array.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+## Example usage:
+## bsub -J "run_fragalysis_retrospective[1-576]" < run_fragalysis_retrospective_array.sh
+
+#BSUB -J run_oe_docking
+#BSUB -oo run_fragalysis_retrospective_%I.out
+#BSUB -eo run_fragalysis_retrospective_%I.stderr
+#BSUB -n 32
+#BSUB -q cpuqueue
+#BSUB -R rusage[mem=3]
+#BSUB -W 2:00
+source ~/.bashrc
+conda activate ad-3.9
```

Comment:
this will break for other users of course, perhaps best to put in a comment to point the users to their own conda envs

Reply:
this is true for all the cluster scripts...i'll make a separate issue for this, maybe there's a better way to keep these in the repo? i'd appreciate keeping them

Reply:
Added issue here https://github.com/choderalab/asapdiscovery/issues/230